satpy/tests/reader_tests/test_agri_l1.py

```diff
@@ -0,0 +1,540 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2019 Xin Zhang
+
+# Author(s):
+
+#   Xin Zhang <xinzhang1215@gmail.com>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Like the reader module, check other modules to see the now standardized license header that we include in all modules.